src/commands.js

```diff
@@ -2,26 +2,30 @@ import { exec } from "child_process";
 import chalk from "chalk";
 import { logExec } from "./helper.js";
 
-export async function getStaggedFiles() {
-  return exec("git diff --cached --name-only", (err, stdout, stderr) => {
-    if (err) {
-      logExec(err);
-      process.exit(0);
-    }
-    if (stderr) {
-      logExec(stderr);
-      process.exit(0);
-    }
+export async function getStagedFiles() {
+  return new Promise((resolve, reject) => {
+    exec("git diff --cached --name-only", (err, stdout, stderr) => {
+      if (err) {
+        logExec(err);
+        reject(err);
+      }
+      if (stderr) {
+        logExec(stderr);
+        reject(stderr);
+      }
 
-    if (stdout.length <= 0) {
-      console.log(chalk.bgRed("⛔️ Whops! No files available to commit."));
-      console.log(
-        "💡 Make sure to use " +
-          chalk.bgWhite("git add") +
-          " before trying to commit."
-      );
-      process.exit(0);
-    }
+      if (stdout.length <= 0) {
+        console.log(chalk.bgRed("⛔️ Whops! No files available to commit."));
+        console.log(
+          "💡 Make sure to use " +
+            chalk.bgWhite("git add") +
+            " before trying to commit."
+        );
+        resolve(null);
+      } else {
+        resolve(stdout.trim().split("\n"));
```

Comment:
We are not using the data anywhere. We don't need to split the files, sending `stdout` as it is should also work.

Reply:
we don't right now, I was making it more readable because I had something else in mind, it is better to let the user know which files are staged, so they don't mistakenly commit wrong files, but we can remove it for now